pkg/epinio/components/application/AppConfiguration.vue

```diff
@@ -71,6 +70,11 @@ export default Vue.extend<Data, any, any, any>({
         .filter((s: EpinioService) => s.metadata.namespace === this.application.metadata.namespace);
     },
 
+    namespacedConfigurations() {
+      return (this.application.allConfigurations || [])
+        .filter((s: EpinioService) => s.metadata.namespace === this.application.metadata.namespace);
```

Comment:
`application.allConfigurations` already filters by namespace

Reply:
The bug is coming from `allConfigurations` , where we are filtering based on namespace and configuration name from app's configuration:

```
.filter((s) => {
  return s.metadata.namespace === this.metadata.namespace &&
   this.configuration.configurations.find(c => c === s.metadata.name);
});
```

So just removing all configuration and filtering only by namespace.

Reply:
ahhh, sorry, i get it